### PR TITLE
[CSP-245] Update for enforce sign-in UI

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes
Do not merge until Sept. 19

- Inform user that they can use Docker Hub to download the registry.json file or copy the specific commands to create it for an organization.

### Related issues (optional)
CSP-245

### CODE REVIEW

**Review by marisimon — COMMENTED** [review]
Nice work! Left one question.

File: _includes/configure-registry-json.md [hunk 1]
```diff
@@ -23,7 +23,9 @@ The `registry.json` file must contain the following contents, where `myorg` is r
 }
 ```
 
-You can use the following methods to create a `registry.json` file based on the user's operating system.
+In Docker Hub, you can download the registry.json file for your organization or copy the specific commands to create the file for your organization. To download the file or copy the commands from [Docker Hub](http://hub.docker.com), go to **Organizations** > **_Your Organization_** > **Settings** > **Enforce Sign-in**.
```
Comment (jerae-duffin) [inline _includes/configure-registry-json.md:26]: When the Admin selects the Settings tab, is the Enforce Sign-in located in the left navigation? Is it a button in the UI?

Comment (craig-osterhout) [reply _includes/configure-registry-json.md:26]: In the left navigation.

File: docker-hub/release-notes.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ known issues for each Docker Hub release.
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## 2022-09-19
+### Enhancement
+
+In Docker Hub, you can now download a [registry.json](../docker-hub/configure-sign-in.md) file for your organization or copy the commands to create a registry.json file for your organization.
```
[the remark this answers is not in the document]
Comment (craig-osterhout) [reply docker-hub/release-notes.md:18]: Thanks, Mari. The content from ` configure-registry-json.md` is included in [configure-sign-in.md](https://deploy-preview-15637--docsdocker.netlify.app/docker-hub/configure-sign-in/).
It's shared content that appears in a few topics. For example, it also appears in the [Image Access Mangement](https://deploy-preview-15637--docsdocker.netlify.app/docker-hub/image-access-management/#create-a-registryjson-file) topic.

File: _includes/configure-registry-json.md (hunk 1 above)
Comment (jerae-duffin) [reply _includes/configure-registry-json.md:26]: Suggestion: 
 In Docker Hub, you can download the registry.json file for your organization or copy the specific commands to create the file for your organization. To download the file or copy the commands from [Docker Hub](http://hub.docker.com), go to **Organizations** > **_Your Organization_** > **Settings** . 

Select Enforce Sign-in and continue with the on-screen instructions for Windows, Mac, or Linux.

Comment (craig-osterhout) [reply _includes/configure-registry-json.md]: Thanks, @jerae-duffin. I separated it into ordered steps in the following commit. PTAL

File: docker-hub/release-notes.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ known issues for each Docker Hub release.
...
```
Comment (marisimon) [inline docker-hub/release-notes.md:17]: Maybe worth adding the goal of enforcing sign-in:

Suggestion:
 In Docker Hub, you can now download a [registry.json](../docker-hub/configure-sign-in.md) file or copy the commands to create a registry.json file **to enforce sign-in** for your organization.

**Review by marisimon — APPROVED** [review]
LGTM! Left one suggestion.

**Review by jerae-duffin — DISMISSED** [review]
LGTM!

File: docker-hub/release-notes.md (hunk 3 above)
Comment (craig-osterhout) [reply docker-hub/release-notes.md]: Thanks, @marisimon. It's been updated in a following commit. PTAL

**Review by jerae-duffin — APPROVED** [review]
LGTM!

**Pull request merged**